### PR TITLE
Milestone 2 : Using Bootstrap (Newsweek)

### PR DESCRIPTION
**Pull request by ZeenatLawal**
## Newsweek clone milestone 2: Using Bootstrap
In this milestone we:  

- Added 3 columns main section of the website and adapted for the 3 screen sizes
- Used bootstrap's default media query breaking points (Tablet >= 768px, Desktop >= 992px) 
- Used bootstrap classes for the styling

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,15 +59,956 @@
...
+                            <h5><a href="#" class="fw-bold link-dark text-decoration-none">After Parler Ban, Rein in Big Tech Now or Cease Being Free Citizens</a></h5>
+                            <span>BY <a href="#" class="link-secondary text-decoration-none">RACHEL BOVARD</a></span>
+                        </div>
+                    </article>
+                </section>
+                <div class="gray w-100 d-md-none"> <br></div>
+                <!-- OPINION -->
+                <section class="d-md-none">
+                    <h5 class="text-secondary">OPINION</h5>
+                    <article class="border-bottom border-2 mb-3 pb-2">
```
Comment (Dipeshtwis) [inline index.html:200]: All the titles should have some more font-weight so that it looks the same.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status:  ♻️ Changes Required ♻️
 Hi @ZeenatLawal  and @safafa

**You've really hit all the requirements in one shot. Well done.** Here're the major highlights of this PR:
- Your PR title and description are really descriptive.
- The content order and disposition changes to the correct place on the desktop, tablet, and mobile.

**However, in order to approve, we have to work a little bit more. Please check my comments below and fix them as appropriate.**

## For Approval

- [x] All images should have links.
- [x] All the titles should have a link and more font-weight.

- [x] The free sign-up text should be aligned. I have made the border.

Screenshot (1303)_LI

- [x] This sponsored insight is breaking in 992px to 1087px. What I will suggest you to reduce the font size.

Screenshot (1304)_LI

## [Optional]

- [x] I will recommend giving a little bit more gap from the side. It's okay as well if you don't give.

Screenshot (1305)_LI

Remember, This is optional and nothing to do with the project approval.

## Please make the required changes and submit another review request when you are done.

## Reminder

**"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) now, you have only 2 code review left for this project."**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Twitter](https://twitter.com/97deepeshkumar)

File: style.css [hunk 2]
```diff
@@ -41,7 +48,68 @@ footer .fab:hover {
...
+  bottom: -16px;
+  z-index: 1;
+  width: 32px;
+  height: 32px;
+  background-color: #000;
+  color: #d6d6d6;
+  border-radius: 50%;
+  text-align: center;
+  font-weight: bold;
+}
```
Comment (Berabjesus) [inline style.css:84]: - [ ] [OPTIONAL] avoid using rules that you can already apply using bootstrap.
- In this case `position: absolute;`, ` text-align: center;`, `font-weight: bold;` can be used with bootstrap

File: index.html [hunk 3]
```diff
@@ -53,15 +59,958 @@
...
+                        </div>
+                    </article>
+                </section>
+                <!-- MY TURN -->
+                <section>
+                    <a href="#" class="text-decoration-none"><h5 class="text-secondary fw-bold mt-3 ms-md-0">MY TURN</h5></a>
+                    <div class="d-flex flex-column flex-md-row font-medium">
+                        <div>
+                            <a href="#"><img src="image/middle.png" class="img-fluid pe-md-2" alt="..."></a>
+                            <h5 class="text-secondary my-4 my-md-2">'I Married My Best Friend to Avoid Inheritance Tax' </h5>
```
Comment (Berabjesus) [inline index.html]: - [x] All the titles should have a link and more font-weight. (Each title uses a hx tag and has a link) - Check the comment below. As you can see on the original website, this and all the other similar headers are links. So please add the appropriate anchor tag.

**Review by Berabjesus — CHANGES_REQUESTED** [review]
## Status: CHANGES REQUIRED :warning: :construction:

Hi  @ZeenatLawal  and @safafa  You have done really awesome applying the requested changes :+1: ! Awesome README :100: Good job with the professional PR :tada: 

### However, you do have one change to make that was requested in the previous review.

- [x] All the titles should have a link and more font-weight. (Each title uses a hx tag and have a link) - Check the comment below.

**❗ You have one(1) remaining review left. As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), when you reach to the limit of 3 code reviews and your project was not approved, you will need to repeat the project. If you think that the code review was not fair, you can [request a second-opinion using this form.](https://airtable.com/shrmBIv6cXYrq4Fgv)**

#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @ZeenatLawal and @safafa  , your project is approved :tada: :tada: :tada: 

You have implemented all rquested changes from the previous review, so I am marking this project as complete, but before merging all changes to the base branch, I would highly recommend you remove the white space which is on the top of the navbar, looking at the original website, the navbar is touching the top border of the viewport,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**